user-console/src/main/java/org/pentaho/mantle/client/admin/UserDialog.java

```diff
@@ -62,11 +63,18 @@ public UserDialog( UserRolesAdminPanelController controller ) {
 
     acceptBtn.setEnabled( false );
     nameTextBox = new TextBox();
+    Roles.getTextboxRole().set( acceptBtn.getElement() );
+    nameTextBox.getElement().setTabIndex( 0 );
+    nameTextBox.setFocus( true );
```

Comment:
Is `nameTextBox` not the first focusable element in the user dialog? If so, the GwtDialog class shall already place the focus on it (changes on the way by @varuntangirala).

If not the first one, then setting focus explicitly in this way is imo not the best way. A declarative XUL attribute should exist for it. Also, the constructor is not always run as XUL instances are frequently reused...

Reply:
Once the changes from @varuntangirala goes in, I can certainly test and keep whatever is needed and remove what's not needed.